### PR TITLE
Add number of unavailable and total items to shipping-calculator

### PR DESCRIPTION
**Pull request by jeffersontuc**
#### What problem is this solving?

We need to show contextualized warnings about the availability of products to be delivered to a given address. To do that, `shipping-calculator` needs to know the amount of unavailable items as well as the total amount of items added to cart, since there's two types of availability warnings as it's shown below. This PR adds it.

#### How should this be manually tested?

- Proceed to [Workspace](https://deliveryfeedback--checkoutio.myvtex.com/cart/add?sku=308&sku=288)
- Verify that a warning (image 1) it's being shown.
- Remove the available item.
- Verify that another type of warning (image 2) is being shown now.

#### Screenshots or example usage

**Image 1**

image

**Image 2**

image

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] [Linked this PR to a Clubhouse story (if applicable).](https://app.clubhouse.io/vtex/story/27145/implementar-tratamento-de-erros-para-delivery-options)
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/Shipping.tsx [hunk 1]
```diff
@@ -6,8 +6,14 @@ import {
   useOrderShipping,
 } from 'vtex.order-shipping/OrderShipping'
 
+const AVAILABLE = 'available'
```
Comment (darlenedms) [inline react/Shipping.tsx]: If we do it one more time (https://github.com/vtex-apps/checkout-cart/blob/9472282f6a3b87474ce1b9ed2b955e57199368b9/react/Title.tsx#L5), maybe we should create a constants file.

Comment (marcoskwkm) [reply react/Shipping.tsx]: I think 2 uses are already enough to put this in a constants and avoid future bugs.

Comment (jeffersontuc) [reply react/Shipping.tsx]: gotcha!

File: react/consts.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const AVAILABLE = 'available'
```
Comment (marcoskwkm) [inline react/consts.ts]: nit: I'd rename this to `constants.ts` to maintain consistency across other VTEX repos.

**Pull request merged**